### PR TITLE
Fix default value of compose --registry flag

### PR DESCRIPTION
**Pull request by Lothiraldan**

### CODE REVIEW

File: taskboot/cli.py [hunk 1]
```diff
@@ -93,7 +93,12 @@ def main():
         help='Path to docker-compose.yml to use',
         default='docker-compose.yml',
     )
-    compose.add_argument('--registry', type=str, help='Docker registry to use in images tags')
+    build.add_argument(
```
Comment (marco-c) [inline taskboot/cli.py]: Should be `compose`

Comment (La0) [inline taskboot/cli.py]: `s/build/compose/`

File: taskboot/cli.py [hunk 2]
```diff
@@ -93,7 +93,12 @@ def main():
         help='Path to docker-compose.yml to use',
         default='docker-compose.yml',
     )
-    compose.add_argument('--registry', type=str, help='Docker registry to use in images tags')
+    compose.add_argument(
+        '--registry',
+        type=str,
+        default=os.environ.get('REGISTRY', 'registry.hub.docker.com'),
```
Comment (marco-c) [inline taskboot/cli.py]: Maybe use the same default in the build --registry argument?

Comment (La0) [reply taskboot/cli.py]: good point

Comment (Lothiraldan) [reply taskboot/cli.py]: Sure, making a commit right now

**Pull request merged**